samples/HelloWorld/HelloWorld.al

```diff
@@ -11,6 +11,6 @@ codeunit 70051001 HelloWorld
     var
         HelloText : Codeunit GreetingsManagement;
     begin
-        Message(HelloText.GetRandomGreeting() + ', ' + Rec.Name);
+        Message('%1, %2',HelloText.GetRandomGreeting(),Rec.Name);
```

Comment:
I like the PowerShell-way of being able to use Variables in strings.  Something like:
`Message("$(HelloText.GetRandomGreeting()), $(Rec.Name)")`